src/peep/peep.c

```diff
@@ -1089,11 +1105,17 @@ static void sub_68F41A(rct_peep *peep, int index)
 						possible_thoughts[num_thoughts++] = PEEP_THOUGHT_TYPE_BATHROOM;
 					}
 
-					// Not sure why the happiness check is like that seems wrong to me
 					if (!(gParkFlags & PARK_FLAGS_NO_MONEY) &&
 						peep->cash_in_pocket <= MONEY(9, 00) &&
 						peep->happiness >= 105 &&
-						peep->happiness >= 70){
+						peep->energy >= 70){
```

Comment:
That particular change is related to the running out of money thought - commit 43ef6d9.
The comparison I'm doing is related to commit cbc8db0.  If you have some way to check this easily, that would be fantastic!